dotcom-rendering/src/web/components/App.tsx

```diff
@@ -324,6 +325,27 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 				}
 			});
 
+			onConsentChange((consentState) => {
+				const decideConsentString = () => {
+					if (consentState.tcfv2) {
+						return consentState?.tcfv2?.tcString;
```

Comment:
Note, you have a typo here - t`c`String -> t`o`String.

Reply:
@nicl 

Actually it's `tcString`

<img width="804" alt="Screenshot 2021-09-03 at 16 02 43" src="https://user-images.githubusercontent.com/6035518/132026870-8a210be3-7824-4935-a3bb-afda5f92bf9c.png">

Reply:
wat. Okay thanks!

Reply:
As in https://iabtcf.com/api/core/classes/tcstring.html